Scripts/extras/pull_and_build_from_git.py

```diff
@@ -751,12 +758,21 @@ def prepare_build(platform_name, base_folder, build_folder, package_root_folder,
     :param toolchain_file:      Option toolchain file to use for specific target platforms
     :param build_config_file:   The build config file to open from the base_folder
     :param clean:               Option to clean any existing build folder before proceeding
+    :param src_folder:          Option to manually specify the src folder
+    :param skip_git:            Option to skip all git commands, requires src_folder be supplied
```

Comment:
I think skip_git is implicitly True if src_folder is supplied, right? We may not need this

Reply:
I didn't want to assume that supplying the code folder meant that no git commands should be run. It could just be that they want the repo cloned to a specific folder.